VERSION

```diff
@@ -1 +1 @@
-0.0.12
\ No newline at end of file
+0.0.13
```

Comment:
This is a major version change, as renames are breaking changes.

Reply:
It is, but no one is using this library.